### PR TITLE
Additions and Revisions

### PR DESCRIPTION
**Pull request by johnpierson**
- Revise copyright year,
- Addition of space analysis package section
- reorganization

### CODE REVIEW

**Comment by johnpierson** [comment]
This is in progress, FYI @lillismith @mrahmaniasl

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Let us know about any issues with this document.
 
 ## License
 
-Copyright 2019 Autodesk
+Copyright © 2020 Autodesk
```
Comment (lillismith) [inline README.md:45]: Let's go ahead and change it to 2021

Comment (johnpierson) [reply README.md]: Whoops. That was definitely a result of me forgetting it is 2021. 🤣

**Comment by johnpierson** [comment]
@lillismith this first batch of changes should be good to go.

File: 03-hello-GD-For-Revit/03-05_refinery-toolkit/03-05-02_using-the-refinery-toolkit.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Using the Refinery Toolkit
+The package comes with a number of sample files provided to help you get started with each of the libraries.
+
+It is highly recommended to start with the samples as they contain detailed notes and instructions on how to use each of the nodes. Feel free to open an issue or submit a PR on the [GitHub repo](https://github.com/DynamoDS/RefineryToolkits) if you’d like to see further documentation added. 
+
+You can find the sample files in the package repository’s samples folder, as well as in the extra folder of the package when downloaded through the Dynamo Package Manager, typically found here: `%appdata%\Dynamo\Dynamo Revit\2\packages\Refinery Toolkit`
+
+The samples show simple examples of how each node in the toolkit works. Some of the samples have a Revit version (marked with `(RVT)`) and some also have a version to be used directly with Generative Design in Revit (marked with `(RefineryVersion)`).
+
```
Comment (lillismith) [inline 03-hello-GD-For-Revit/03-05_refinery-toolkit/03-05-02_using-the-refinery-toolkit.md:9]: What do you mean by "Refinery Version" ?

Comment (johnpierson) [reply 03-hello-GD-For-Revit/03-05_refinery-toolkit/03-05-02_using-the-refinery-toolkit.md:9]: Hmm, that is something weird with the samples and should probably be removed. I think the idea was to indicate something that worked with `Core` vs `Revit` context. 
image

**Pull request merged**